pkg/logutil/log.go

```diff
@@ -292,3 +293,65 @@ func LogPanic() {
 		zaplog.Fatal("panic", zap.Reflect("recover", e))
 	}
 }
+
+var (
+	enabledRedactLog atomic.Value
```

Comment:
how about using `atomic.Bool` from `go.uber.org/atomic`?

Reply:
`go.uber.org/atomic` is good, but I think it's better to unify the `atomic` introducing in PD.